Makefile

```diff
@@ -1036,3 +1039,89 @@ ceph_cleanup: ## deletes the ceph pod
 	$(eval $(call vars,$@,ceph))
 	oc kustomize ${DEPLOY_DIR} | oc delete --ignore-not-found=true -f -
 	rm -Rf ${DEPLOY_DIR}
+
+##@ NMSTATE
+.PHONY: nmstate
+nmstate: export NAMESPACE=openshift-nmstate
+nmstate: ## installs nmstate operator in the openshift-nmstate namespace
+	$(eval $(call vars,$@,nmstate))
+	bash scripts/gen-namespace.sh
+	oc apply -f ${OUT}/${NAMESPACE}/namespace.yaml
+	sleep 2
+	bash scripts/gen-olm-nmstate.sh
+	oc apply -f ${OPERATOR_DIR}
+	while ! (oc get pod --no-headers=true -l app=kubernetes-nmstate-operator -n ${NAMESPACE}| grep "nmstate-operator"); do sleep 10; done
+	oc wait pod -n ${NAMESPACE} --for condition=Ready -l app=kubernetes-nmstate-operator --timeout=180s
```

Comment:
timeout was too short for my local deployment

Reply:
ok, I bumped them to be 300s, as its a max it does not hurt